src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-notification-list/rgw-bucket-notification-list.component.html

```diff
@@ -0,0 +1,35 @@
+  <fieldset>
+    <legend i18n>
+    Notification Configuration
+    <cd-help-text>
+      Create a notification configuration to trigger alerts for specific bucket events, such as object creation or transitions, based on prefixes or tags.
+    </cd-help-text>
```

Comment:
<img width="1074" height="96" alt="Screenshot From 2025-07-14 12-55-13" src="https://github.com/user-attachments/assets/952ed3fb-9e69-4d00-803a-f64b01f1913f" />

can we reduce the font size use h1 headings and carbon type stylings ?